pkg/verifier/codeowner.go

```diff
@@ -183,6 +190,17 @@ func getPatternFromLine(line string) (*regexp.Regexp, bool) {
 	return pattern, negatePattern
 }
 
+func FilePathWalkDir(root string) ([]string, error) {
+	var files []string
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
```

Comment:
It would be better if we verify the received error on the argument and break the execution:

```golang
 	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error { 
            if err != nil {
                return err
            }
            ...
        })